File: cms_server/cms_server/Entities/AdminEntity.cs
using Microsoft.AspNetCore.Http;
using System;

namespace cms_server.Entities
{
    public class AdminEntity
    {
        public int? admin_pk { get; set; }
        public int? user_pk { get; set; }
        public string emp_id { get; set; }
        public string pic_dest { get; set; }
        public string first_name { get; set; }
        public string middle_name { get; set; }
        public string last_name { get; set; }
        public string suffix { get; set; }
        public string gender { get; set; }
        public string position { get; set; }
        public string email { get; set; }
        public string mob_no { get; set; }
        public string is_active { get; set; }
        public DateTime? encoded_at { get; set; }
        public string encoder_pk { get; set; }
        //ext
        public IFormFile img_attach { get; set; }
    }
}

File: cms_web/src/Services/Entities/AdminEntity.ts
interface AdminEntity {
  admin_pk?: number;
  user_pk?: number;
  emp_id?: string;
  pic_dest?: string;
  first_name?: string;
  middle_name?: string;
  last_name?: string;
  suffix?: string;
  gender?: "m" | "f";
  position?: string;
  email?: string;
  mob_no?: string;
  is_active?: "y" | "n";
  encoded_at?: Date;
  encoded_by?: string;

  //ext
  img_attach?: any;
}

export interface AdminTableModel {
  limit: number;
  count: number;
  begin: number;
  table: Array<AdminEntity>;
}

export default AdminEntity;
